server/src/main/scala/org/http4s/server/staticcontent/FileService.scala

```diff
@@ -47,8 +47,8 @@ object FileService {
   /* Returns responses for static files.
    * Directories are forbidden.
    */
-  private def filesOnly(file: File, config: Config, req: Request): Task[Option[Response]] = Task.now {
-    if (file.isDirectory()) Some(Response(Status.Unauthorized))
+  def defaultFileCollector(file: File, config: Config, req: Request): Task[Option[Response]] = Task.now {
+    if (file.isDirectory) StaticFile.fromFile(new File(file, "index.html"), Some(req))
```

Comment:
What is generated if the file does not exist? As this now generates `None` rather than unauthorized. What would be returned? Could we fold our option into a guaranteed response here?